job-service-db/src/main/resources/v3.3.0/changelog.xml

```diff
@@ -23,10 +23,16 @@
                                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
                                        http://www.liquibase.org/xml/ns/dbchangelog
                                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
-    <changeSet id="add_identity_column" runOnChange="true" author="Gamma Team">
-        <addColumn schemaName="public" tableName="job">
-            <column name="identity" type="SERIAL">
-                <constraints nullable="false" />
+
+    <changeSet id="update_job_task_data_add_suspended" runOnChange="true" author="Gamma Team">
+        <preConditions onFail="MARK_RAN">
```

Comment:
What is this and why is it necessary?  Why isn't this identical to [v3.3.0/changelog.xml](https://github.com/JobService/job-service/blob/v3.3.0-source/job-service-db/src/main/resources/v3.3.0/changelog.xml#L27)?

Reply:
To skip column creation if upgrading from version 3.3 which would already have the 3.3.0 changes applied.

Reply:
But surely skipping the changeset if it's already been applied is default behavior for Liquibase?